### DISCUSSION TITLE
if you add watch to one repo, all issues was marked as subscribed

### DISCUSSION THREAD

**Discussion by heartacker** [community/community#27764] [Mobile] [answered]
1. add watch to a repo, like new release
2. use github mobile to view the repo issues.

all issucs are marked as subscribed by mistake

**Comment by jfullstackdev** [comment]
check this out 

https://www.wired.com/2012/08/github-changes-make-it-easier-to-track-your-favorite-projects/

**Reply by heartacker** [reply]
no, this is a bug of github-mobile

**Reply by heartacker** [reply]
I meas that ， I just add `new release` to watch.

and why do any issues is being subscribed in github mobile
for example:
I donot subnscibe the issue,
![image](https://user-images.githubusercontent.com/1876302/182013621-86a4505c-5f86-4751-a7ff-709108a20ac8.png)

but in github mobile,
It is show that I Have subscribed，and the action is become unsubscribe
![ab8d036604986369a55b84091a8024c](https://user-images.githubusercontent.com/1876302/182013690-2cacb115-a76c-438d-9785-f17b5fc79312.jpg)
@jdevstatic

**Reply by jfullstackdev** [reply]
you mean, you just watch the releases but all issues are subscribed ?

**Reply by jfullstackdev** [reply]
I just tried it by now 

![image](https://user-images.githubusercontent.com/47092464/182023354-657e4870-45ca-4682-ab66-a5bccd344ad2.png)

but issues are not subscribed

![image](https://user-images.githubusercontent.com/47092464/182023376-7132e1d0-11a4-4413-ae8e-e83ff6f08df1.png)

**Reply by heartacker** [reply]
> I just tried it by now 
> 
> ![image](https://user-images.githubusercontent.com/47092464/182023354-657e4870-45ca-4682-ab66-a5bccd344ad2.png)
> 
> but issues are not subscribed
> 
> ![image](https://user-images.githubusercontent.com/47092464/182023376-7132e1d0-11a4-4413-ae8e-e83ff6f08df1.png)
> 

Could you please use  github mobile?
This is issue of github mobile

**Reply by heartacker** [reply]
ues github mobile to see that issue

**Reply by jfullstackdev** [reply]
unfortunately I don't use mobile for GitHub, so hopefully this will be seen by Staff members to fix this issue

**Comment by stevepopovich** [comment]
@heartacker I think I understand the problem. Are you actually receiving notifications for these issues? I.e. you subscribe to watch for releases, but then are getting notifications for issues?

I believe this to be a UI issue in which we shouldn't be showing `unsubscribe` in that menu, but in reality, you are not subscribed to any specific issues.

Can you share your GitHub version and what platform this is happening on?

**Reply by heartacker** [reply]
I believe that is the UI issue，
this bug was added since the day that github mobile hides the subscribe/unsubscribe button to menu

**Reply by heartacker** [reply]
Android

**Reply by heartacker** [reply]
GitHub Android v1.75.0; OS SDK v27; smartisan DE106

**Comment by stevepopovich** [comment, marked as answer]
I reported this to the team for prioritization. Thanks for reporting!

**Reply by heartacker** [reply]
When can we fix this? Is there any update?

**Reply by stevepopovich** [reply]
This is in our backlog to be fixed. We will message here when we fix it!

**Reply by meyfa** [reply]
> This is in our backlog to be fixed. We will message here when we fix it!

@stevepopovich Is there any fix in sight? The GitHub app is great, but this particular bug bothers me a lot, since I have to switch to a browser every time I'd like to subscribe to an issue on a repo with custom notifications.

**Reply by stevepopovich** [reply]
Sorry, we have not been able to put this in our timeline yet. Thanks for the interest though!
